gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/LeaseAttemptStatus.java

```diff
@@ -57,6 +58,8 @@ public static class NoLongerLeasingStatus extends LeaseAttemptStatus {}
   current LeaseObtainedStatus via the completeLease method from a caller without access to the {@link MultiActiveLeaseArbiter}.
   */
   @Data
+  // avoid - warning: Generating equals/hashCode implementation but without a call to superclass, even though this class does not extend java.lang.Object
```

Comment:
another preventable source of compilation warnings

Reply:
if it is to prevent using any future fields added to `LeaseAttemptStatus`, then why not add `callSuper=false` to NoLongerLeasingStatus?

Reply:
warnings at least don't indicate that it's necessary over there and I believe that's because warning arose since `LeaseObtainedStatus` and `LeasedToAnotherStatus` are `@Data`, whereas `NoLongerLeasingStatus` is not.

since not `@Data`, lombok is not synthesizing `equals` for `NoLongerLeasingStatus`